<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MYYOLO SYSTEM</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f0f0f0;
}
QMenuBar {
    background-color: #2c3e50;
    color: white;
}
QMenuBar::item:selected {
    background-color: #34495e;
}
QMenu {
    background-color: #2c3e50;
    color: white;
}
QMenu::item:selected {
    background-color: #34495e;
}
QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    padding: 5px 15px;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #2980b9;
}
QPushButton:pressed {
    background-color: #1c6ea4;
}
QComboBox {
    border: 1px solid #bdc3c7;
    border-radius: 4px;
    padding: 5px;
    background-color: white;
}
QSlider::groove:horizontal {
    border: 1px solid #bdc3c7;
    height: 8px;
    background: #ecf0f1;
    margin: 2px 0;
    border-radius: 4px;
}
QSlider::handle:horizontal {
    background: #3498db;
    border: 1px solid #2980b9;
    width: 18px;
    margin: -2px 0;
    border-radius: 9px;
}
QProgressBar {
    border: 1px solid #bdc3c7;
    border-radius: 4px;
    text-align: center;
    background-color: #ecf0f1;
}
QProgressBar::chunk {
    background-color: #3498db;
    border-radius: 3px;
}
QTextEdit {
    border: 1px solid #bdc3c7;
    border-radius: 4px;
    padding: 5px;
    background-color: white;
}
QGroupBox {
    border: 1px solid #bdc3c7;
    border-radius: 4px;
    margin-top: 10px;
    padding-top: 15px;
    background-color: white;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px;
    background-color: #3498db;
    color: white;
    border-radius: 2px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QMenuBar" name="menuBar">
      <widget class="QMenu" name="menuFile">
       <property name="title">
        <string>文件</string>
       </property>
      </widget>
      <widget class="QMenu" name="menuEdit">
       <property name="title">
        <string>编辑</string>
       </property>
      </widget>
      <widget class="QMenu" name="menuHelp">
       <property name="title">
        <string>帮助</string>
       </property>
      </widget>
      <addaction name="menuFile"/>
      <addaction name="menuEdit"/>
      <addaction name="menuHelp"/>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="menuList">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget {
    background-color: #2c3e50;
    color: white;
    border: none;
    border-radius: 4px;
}
QListWidget::item {
    padding: 10px;
    border-bottom: 1px solid #34495e;
}
QListWidget::item:selected {
    background-color: #3498db;
}</string>
      </property>
      <item>
       <property name="text">
        <string>图片检测</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>视频检测</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>摄像头检测</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>批量处理</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>数据库</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="inputGroup">
      <property name="title">
       <string>输入画面</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="inputLabel">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #ecf0f1;
    border: 2px dashed #bdc3c7;
    border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string>输入画面</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="outputGroup">
      <property name="title">
       <string>检测结果</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="outputLabel">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #ecf0f1;
    border: 2px dashed #bdc3c7;
    border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string>检测结果</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="controlGroup">
      <property name="title">
       <string>控制面板</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>开始检测</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>停止检测</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QGroupBox" name="settingsGroup">
      <property name="title">
       <string>参数设置</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="modelLabel">
         <property name="text">
          <string>模型选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="modelComboBox">
         <item>
          <property name="text">
           <string>YOLOv5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YOLOv8</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="confidenceLabel">
         <property name="text">
          <string>置信度阈值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="confidenceSlider">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="confidenceValueLayout">
         <item>
          <widget class="QSpinBox" name="confidenceSpinBox">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="confidenceValueLabel">
           <property name="text">
            <string>50%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="iouLabel">
         <property name="text">
          <string>IOU阈值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="iouSlider">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="iouValueLayout">
         <item>
          <widget class="QSpinBox" name="iouSpinBox">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="iouValueLabel">
           <property name="text">
            <string>50%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lineWidthLabel">
         <property name="text">
          <string>线宽</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="lineWidthSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="lineWidthValueLayout">
         <item>
          <widget class="QSpinBox" name="lineWidthSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lineWidthValueLabel">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>保存结果</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QTextEdit" name="messageOutput">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>消息输出区域...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
